pkg/apis/secretmanager/v1alpha1/externalsecret_types.go

```diff
@@ -78,16 +78,21 @@ type KeyReference struct {
 }
 
 type RemoteReference struct {
-	// Path to the key in the SecretStore
-	Path string `json:"path"`
+	// Path to the key in the SecretStore. Must be a supported parameter
+	// +optional
+	Path *string `json:"path"`
```

Comment:
These probably should be merged into a single field like `name`. Feels sufficiently generic enough to handle both path based access like vault, or name/id's in other backends.